modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/tracing/opentelemetry/management/helpers/SpanTagger.java

```diff
@@ -42,67 +42,67 @@ private SpanTagger() {}
     public static void setSpanTags(SpanWrapper spanWrapper) {
         StatisticsLog openStatisticsLog = new StatisticsLog(spanWrapper.getStatisticDataUnit());
         Span span = spanWrapper.getSpan();
-        if (OpenTracingManagerHolder.isCollectingPayloads() || OpenTracingManagerHolder.isCollectingProperties()) {
-            if (OpenTracingManagerHolder.isCollectingPayloads()) {
+        if (OpenTelemetryManagerHolder.isCollectingPayloads() || OpenTelemetryManagerHolder.isCollectingProperties()) {
+            if (OpenTelemetryManagerHolder.isCollectingPayloads()) {
                 if(openStatisticsLog.getBeforePayload() != null) {
-                    span.setTag("beforePayload", openStatisticsLog.getBeforePayload());
+                    span.setAttribute("beforePayload", openStatisticsLog.getBeforePayload());
```

Comment:
Shall we create a new class called `SpanTaggerConstants`, and introduce constants for all these tag keys?

Reply:
Resolved after introducing constatnts to TelemetryConstants class.